Userland/Libraries/LibWeb/Layout/BlockFormattingContext.cpp

```diff
@@ -462,18 +462,25 @@ void BlockFormattingContext::resolve_used_height_if_treated_as_auto(Box const& b
         height = max(height, calculate_inner_height(box, available_space.height, computed_values.min_height()));
     }
 
+    bool has_vertical_writing_mode = box.computed_values().writing_mode() == CSS::WritingMode::VerticalLr || box.computed_values().writing_mode() == CSS::WritingMode::VerticalRl;
+    bool has_horizontal_writing_mode = box.computed_values().writing_mode() == CSS::WritingMode::HorizontalTb;
```

Comment:
It feels a bit wasteful to call `box.computed_values().writing_mode()` 3 times. Also, the writing mode is always either horizontal or vertical, so one can be defined as the opposite of the other, or even just as one variable.